src/Common/src/CoreLib/System/Memory.cs

```diff
@@ -12,6 +12,10 @@
 
 namespace System
 {
+    /// <summary>
+    /// Memory represents a contiguous region of arbitrary memory similar to Span.
```

Comment:
nit: Use decorator tag cref when referring to Span, similar to the comment in ReadOnlyMemory.cs

    /// <summary>
    /// Represents a contiguous region of memory, similar to <see cref="Span{T}"/>.
    /// Unlike <see cref="Span{T}"/>, it is not a byref-like type.
    /// </summary>